lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ShowPageProducer.java

```diff
@@ -4616,6 +4616,13 @@ private void createAnnouncementsDialog(UIContainer tofill, SimplePage currentPag
 	// in the edit dialogs
 	private void createAddMultimediaDialog(UIContainer tofill, SimplePage currentPage) {
 		UIOutput.make(tofill, "add-multimedia-dialog").decorate(new UIFreeAttributeDecorator("title", messageLocator.getMessage("simplepage.resource")));
+
+		String max = ServerConfigurationService.getString("content.upload.max", 20);
+		String ceiling = ServerConfigurationService.getString("content.upload.ceiling", max);
+		String uploadMax = ceiling;
```

Comment:
tabs vs spaces.

Reply:
This does not compile, 

String max = ServerConfigurationService.getString("content.upload.max", "20");